contact.html

```diff
@@ -285,6 +291,10 @@ <h3>Ubícanos:</h3>
 		<script src="js/validations.js"></script>
 		<!-- Fonts awesome for validations texts -->
 		<script src="https://kit.fontawesome.com/2c36e9b7b1.js" crossorigin="anonymous"></script>
+		<?php
```

Comment:
PHP file doesn't exist and file extension is HTML.

Reply:
It was resolve